driver-core/src/main/com/mongodb/connection/ResponseBuffers.java

```diff
@@ -41,6 +43,12 @@ public ReplyHeader getReplyHeader() {
         return replyHeader;
     }
 
+    <T extends BsonDocument> T getResponseDocument(final int messageId, final Decoder<T> decoder) {
```

Comment:
Moved here from InternalStreamConnection so that it can be used by LoggingCommandEventSender as well.

Reply:
Ack